updated querystring functionName= binding

MelvinVermeer opened the pull request:
The Azure Functions CLI 1.0.9 handles the FunctionName attribute correctly. So this removed statement was no longer valid.

azurecla commented:
Hi __@NarayanAnnamalai__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

joaoma commented:
@tysonn  please merge. this is an important change in the doc set.

Commit:
updated querystring functionName= binding

The Azure Functions CLI 1.0.9 handles the FunctionName attribute correctly. So this removed statement was no longer valid.
